### ISSUE TITLE
How to incorporate ratings while building the dataset

### ISSUE THREAD

**Issue by hugos0910** [lyst/lightfm#345]
Hi, I'm just starting to learn about building recommender systems, so I am attempting to construct my own movielens dataset from raw data downloaded from https://movielens.org/.  

I have been following your dataset building tutorial (Goodbooks-10k), but noticed that the book ratings were not incorporated into the dataset building process.  

Should we incorporate ratings while building the interaction matrix?  Or that's something that comes later?  Thanks

**Comment by maciejkula** [comment]
LightFM doesn't really use ratings. Have a look [here](https://maciejkula.github.io/2018/07/19/dont-use-explicit-feedback-recommenders/) to see why I think ratings are a bad idea.

**Comment by hugos0910** [comment]
Thanks for the response Maciej, I watched the talk and read the paper referenced.

I read on the documentation that LightFM can also handle explicit data.  To do so we just need to choose a suitable loss function?  I will be using implicit feedback for the project I am working on, but just curious about it.

Also, if we scaled the interaction matrix for implicit feedback, will LightFM treat all non-zero values as indicators for interaction and values close to 1 for "stronger" interaction?
Thanks
